### PR TITLE
Add "Enqueue now" button

### LINKED ISSUE #75: Add button for manual enqueue jobs through web ui
I currently migrate from sidetiq gem, which now looks unsupported and sometimes behaves unexpected and I concerned by missing buttons for manually starting jobs through web ui. This feature will be very helpful for development.

### PR DESCRIPTION
**Pull request by reneklacan**
screen shot 2015-12-19 at 14 31 55

Closes #75

### CODE REVIEW

File: web/views/recurring_jobs.erb [hunk 1]
```diff
@@ -24,8 +25,13 @@
             <a href="<%= root_path %>queues/<%= job_spec.fetch('queue', 'default') %>"><%= job_spec.fetch('queue', 'default') %></a>
           </td>
           <td><%= job_spec['args'] %></td>
+          <td class="text-center">
+            <a class="btn btn-warn btn-xs" href="<%= root_path %>recurring-jobs/<%= name %>/enqueue">
```
Comment (elpic) [inline web/views/recurring_jobs.erb]: The name shouldn't be escaped?

Comment (reneklacan) [inline web/views/recurring_jobs.erb]: urls in href are automatically escaped so even though url includes space it's escaped to `%20` etc

**Comment by reneklacan** [comment]
What could be improved on this one is displaying flash like alert after you hit `Enqueue now` button with message like `SuperJob has been enqueued`

File: web/views/recurring_jobs.erb (hunk 1 above)
Comment (reneklacan) [inline web/views/recurring_jobs.erb]: but I will add it just to be 100% sure

**Comment by elpic** [comment]
@reneklacan Sorry for the delay. Could you please squash the commits into one, and rebase with origin? Thanks

File: lib/sidekiq-scheduler/web.rb [hunk 2]
```diff
@@ -10,6 +10,12 @@ def self.registered(app)
 
         erb File.read(File.join(VIEW_PATH, 'recurring_jobs.erb'))
       end
+
+      app.get '/recurring-jobs/:name/enqueue' do
+        schedule = Sidekiq.get_schedule(params[:name])
+        Sidekiq::Scheduler.enqueue_from_config(schedule)
```
Comment (elpic) [inline lib/sidekiq-scheduler/web.rb]: Maybe we can rename this method to enqueue_job. What do you think?

Comment (reneklacan) [inline lib/sidekiq-scheduler/web.rb]: Makes sense :+1:

**Comment by reneklacan** [comment]
@elpic no problem! It's rebased and squashed and I also renamed `enqueue_from_config` to `enqueue_job`

**Pull request merged**